building_energy_standards_data/database_tables/__init__.py

```diff
@@ -87,14 +87,17 @@
     "exterior_lighting_90_1",
     "exterior_lighting_90_1_prm",
     "exterior_lighting_IECC",
+    "support_occupant_energy_behavior",
     # tables with foreign keys
+    "support_occupant_physical_characteristics",
```

Comment:
The `support_occupant_physical_characteristics` do not have foreign keys. Move this line above the comment.

Reply:
Thank you for the comment. I will follow Weili's suggestion at this stage. Originally, the `support_occupant_physical_characteristics` has the following code, which sets `name` in `support_schedules` as the foreign keys. However, this setting cannot pass the test because `name` is not a unique value. Do you have any suggestions to do this right?  

```python
CREATE_PHYSICAL_CHAR_TABLE = """
CREATE TABLE IF NOT EXISTS support_occupant_physical_characteristics (
    id INTEGER PRIMARY KEY,
    physical_characteristic_name TEXT NOT NULL,
    schedule_activity_level TEXT NOT NULL,
    schedule_clothing_insulation TEXT NOT NULL,
    schedule_air_velocity TEXT NOT NULL,
    work_efficiency NUMERIC,
    co2_generation NUMERIC,
    co2_generation_units TEXT,
    annotation TEXT,
    FOREIGN KEY(schedule_activity_level) REFERENCES support_schedules(name),
    FOREIGN KEY(schedule_clothing_insulation) REFERENCES support_schedules(name),
    FOREIGN KEY(schedule_air_velocity) REFERENCES support_schedules(name)
);
```

Reply:
Got it. This is because the Schedule Table schema does not set the name to be Unique because it is not a primary key.
Since this is AOP deliverable, I consider this rather to be a minor issue for now. @jiangyilin123 could you put this to the project issue so we can keep track on it?
@lymereJ the suggestion here is whether we want to enforce name to be UNIQUE **or** ask the support_occupant_physical_characteristics to use the id from support_schedules as the foreign key?

The first option makes the table easier to read
The second option allows duplicated schedule names.